src/battle_script_commands.c

```diff
@@ -17000,3 +17006,34 @@ void BS_TryQuash(void)
     }
     gBattlescriptCurrInstr = cmd->nextInstr;
 }
+
+void BS_CopyFoesStatIncrease(void)
+{
+    NATIVE_ARGS(u8 battler, const u8 *jumpInstr);
+    u32 stat = 0;
+    u32 battler = GetBattlerForBattleScript(cmd->battler);
+
+    if (gQueuedStatBoosts[battler].stats == 0)
+    {
+        memset(gQueuedStatBoosts, 0, sizeof(gQueuedStatBoosts)); // erase all boosts so they are not stacked
+        gBattlescriptCurrInstr = cmd->jumpInstr;    // stats done, exit
+        return;
+    }
+
+    for (stat = 0; stat < (NUM_BATTLE_STATS - 1); stat++)
```

Comment:
This goes through the order Speed -> Sp. Atk -> Sp. Def, no? Probably should fix it so it goes through the correct visual "order"

Reply:
So I'm not sure if this is optimal but I delay the speed check so it is always last